x-pack/test_serverless/functional/es_archives/pre_calculated_histogram/data.json

```diff
@@ -0,0 +1,231 @@
+{
+  "type": "doc",
+  "value": {
+    "id": "index-pattern:histogram-test",
+    "index": ".kibana_analytics_1",
```

Comment:
We try to avoid the usage of system indices in esArchives where possible (outside of specific SO migration tests, this often brings a lot of trouble and has been root cause of race conditions in the past). Have you considered using the `kbnArchiver` to save and restore this data view? Example usage here: https://github.com/elastic/kibana/blob/main/x-pack/test/functional/apps/canvas/custom_elements.ts#L28

Reply:
so it seems this test should be rewritten, but this could als be a follow up PR IMO

Reply:
Thanks for the info! I agree, let's do this work as a followup and fix it in both stateful and serverless.